Wikipedia/Code/PageHistoryCollectionViewCell.swift

```diff
@@ -60,8 +60,7 @@ class PageHistoryCollectionViewCell: CollectionViewCell {
     func updateAccessibilityLabel() {
         let isMinorAccessibilityString = isMinor ? WMFLocalizedString("page-history-revision-minor-edit-accessibility-label", value: "Minor edit", comment: "Accessibility label text used if edit was minor") : ""
         accessibilityLabel = [timeLabel.accessibilityLabel, authorButton.accessibilityLabel, sizeDiffLabel.accessibilityLabel, isMinorAccessibilityString, commentLabel.accessibilityLabel]
-            .compactMap { $0 }
-            .filter { $0.wmf_hasNonWhitespaceText }
+            .compactMap(String.wmf_maybeNonWhitespaceString)
```

Comment:
I'm not sure I see the utility of having this as an extension at the moment given it's only used here. I think I'd leave it inline for now. If we start using this pattern more for generating accessibility labels, I could see the argument for creating an extension to combine all of these steps (removing empty strings from the array and reducing into a comma separated string)

Reply:
no worries - will close for now